modules/developer_manual/pages/app/fundamentals/publishing.adoc

```diff
@@ -310,7 +311,17 @@ code. This does not depend on whether it happens intentionally or not.
 | Max. 4000 characters; Provide all necessary, detailed information about the product. This should contain all user relevant information. Don’t get lost in technical details, focus on the benefits the product offers; Also, use markdown to layout your description.
 
 | license
-| At the moment following license are available: - ocl - ownCloud commercial license; ownCloud enterprise apps only - agpl - mit version The release version of your app. Note: To overwrite a release (using the same version number) it must be in state ``planned``. Once published, you cannot replace a release.
+a| At the moment following license are available:
```

Comment:
Is the "a" at the beginning intended?

Reply:
You may also add at the end of the line after a blank a + so that the text gets written into the same call but next line, else all get concatenated

Reply:
Yes. That's so that the contents of that table cell, which contains AsciiDoc marked up content, will render correctly. Have a read through https://asciidoctor.org/docs/user-manual/#cols-format.